features/FEATURE_LWIP/lwip-interface/lwipopts.h

```diff
@@ -275,4 +275,12 @@
 #error A transport mechanism (Ethernet or PPP) must be defined
 #endif
 
+#include <lwip/arch.h>
+#include "lwip_random.h"
+#include "lwip_tcp_isn.h"
+#define LWIP_HOOK_TCP_ISN lwip_hook_tcp_isn
+#if MBEDTLS_MD5_C
+#define LWIP_USE_EXTERNAL_MBEDTLS 1
```

Comment:
mbed tls c-functions are flagged with MBEDTLS_MD5_C so they do not exists in build without the flag.